canonical/canonical-indico-themes
# Updates css to cover edge cases for the engineering sprint conference

## Done

- Adds style to cover the elements that were not styled for in the original Ubuntu Summit project.
These are:
Date inputs (registration form)
![image](https://user-images.githubusercontent.com/58276363/224957185-26a19adf-5218-4603-965e-c329aa7dafdb.png)
Registration count (homepage)
![image](https://user-images.githubusercontent.com/58276363/224957300-336927fa-941d-4cda-8ea8-b3cdf7449698.png)
Participation list table (participation list)
![image](https://user-images.githubusercontent.com/58276363/224957557-c5c75de6-e9b5-4587-86a8-456e363d62a0.png)

- Fixes some minor visual bugs

## QA

- Setup and run the indico project as described in [the readme](https://github.com/canonical/canonical-indico-themes)
- If you have access to canonicals inidco account, you can log in and render the events as they are on live. If you do not, you will have to create an event and add the associated elements, for example: to get a date input on the registration form, create a registration form and add a input for date. If you have any trouble creating the appropriate elements, please reach out.
- Make sure the highlighted issues have been dealt with and flag any other issue you may find (this plugin was styled specifically for the Ubuntu Summits needs and does not cover every possible usage of the app)

## Issue

Fixes https://warthogs.atlassian.net/browse/WD-2595
